app/src/main/res/layout/bottom_sheet.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/bottom_sheet"
+    android:layout_width="match_parent"
+    android:layout_height="450dp"
+    android:background="@drawable/conu_gradient"
+    android:orientation="vertical"
+    android:padding="5dp"
+    app:behavior_hideable="false"
+    app:behavior_peekHeight="56dp"
+    app:layout_behavior="com.google.android.material.bottomsheet.BottomSheetBehavior"
+    >
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="20dp"
+        android:paddingLeft="192dp"
+        android:orientation="horizontal"
+        android:layout_gravity="center_vertical">
+
+        <!--            android:src="@drawable/up_arrow_foreground"-->
+        <!--        android:textAlignment="center"-->
+        <ImageButton
+            android:id="@+id/arrowBtn"
+            android:layout_width="20dp"
+            android:layout_height="20dp"
+            android:layout_gravity="center_vertical"
+            android:background="?android:selectableItemBackground"
+            />
```

Comment:
put the /> on previous line for consistency, run the formatter

Reply:
reformatted, done